src/zcash/zcashTypes.ts

```diff
@@ -13,6 +13,7 @@ import {
 import type {
   Addresses,
   BalanceEvent,
+  ErrorEvent as ZcashErrorEvent,
```

Comment:
I'm getting `Module '"react-native-zcash"' has no exported member 'ErrorEvent'`

Reply:
That makes sense because this PR is dependent on new versions of rnzcash and rnpirate that don't exist yet